chameleonultragui/lib/gui/component/slot_changer.dart

```diff
@@ -22,7 +22,7 @@ class SlotChangerState extends State<SlotChanger> {
   }
 
   Future<List<Icon>> getFutureData() async {
-    var appState = context.read<MyAppState>();
+    var appState = context.read<ChameleonState>();
```

Comment:
it isnt chameleon state, its AppState or MyAppState and its gonna stay that way. Everyone knows that AppState is the Apps State and ChameleonState could be anything.
Its also a unnsecesary breaking change